frontend/app/src/main/java/dk/scheduling/schedulingfrontend/pages/CreateTaskPage.kt

```diff
@@ -31,126 +32,137 @@ import androidx.compose.ui.Modifier
-                        TimePickerState(0, 0, true),
-                    ),
+        Title(titleText = "Create Task", topMargin = 0.dp)
+
+        Column(
+            modifier =
+                modifier
+                    .fillMaxSize()
+                    .padding(20.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.SpaceBetween,
+        ) {
+            // Dropdown box of devices
+            Spacer(modifier = Modifier.height(100.dp))
+
+            val (task, taskSetter) =
+                remember {
+                    mutableStateOf(
+                        TaskForm(
+                            0,
+                            Duration(""),
+                            DateRange(null, null),
+                            TimePickerState(0, 0, true),
+                            TimePickerState(0, 0, true),
+                        ),
+                    )
+                }
+            var selectedItem by remember { mutableLongStateOf(devices.value.first().id) }
+            StandardDropDownMenu(
+                modifier = Modifier,
+                options = devices.value.associate { device -> device.id to device.name },
+                label = "Devices",
+                selectedItem = selectedItem,
+                onSelect = { selectedItem = it },
+            )
+
+            // Duration input field
+            OutlinedTextField(
+                modifier = modifier.fillMaxWidth(),
+                value = task.duration.value,
+                onValueChange = {
+                    taskSetter(task.copy(duration = Duration(it)))
+                },
+                label = { Text("Duration (min.)") },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                isError = task.duration.isInitializedAndInvalid(),
+            )
+
+            // start date & end date
+            val dateRangeDialog = remember { mutableStateOf(false) }
+
+            StandardDateRangePicker(
+                closeDialog = { dateRangeDialog.value = false },
+                passingDate = {
+                    taskSetter(task.copy(dateRange = it.copy()))
+                },
+                openDialog = dateRangeDialog.value,
+            )
+
+            ClickableCard(
+                { dateRangeDialog.value = true },
+                "Date interval: ${task.dateRange.status().msg}",
+                task.dateRange.isInitialized() && !task.dateRange.status().isValid,
+            )
+
+            // start time
+            val startTimeDialog = remember { mutableStateOf(false) }
+
+            StandardTimePickerDialog(
+                closeDialog = { startTimeDialog.value = false },
+                state = task.startTime,
+                openDialog = startTimeDialog.value,
+            )
+            ClickableCard({ startTimeDialog.value = true }, "Start time: ${task.printStartTime()}")
```

Comment:
```suggestion
            val (startTimeDialog, setStartTimeDialog) = remember { mutableStateOf(false) }

            StandardTimePickerDialog(
                closeDialog = { setStartTimeDialog(false) },
                state = task.startTime,
                openDialog = startTimeDialog,
            )
            ClickableCard({ setStartTimeDialog(true) }, "Start time: ${task.printStartTime()}")
```

This can be done to the other components as well

Reply:
Done